components/ScrollUpButton/index.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { Button } from '@catho/quantum';
+import styles from './ScrollUpButton.module.css';
+
+const ScrollUpButton = () => {
+  const [scrollUpButton, setScrollUpButton] = useState(false);
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY > 50) {
+        setScrollUpButton(true);
+      } else {
+        setScrollUpButton(false);
+      }
```

Comment:
Perfect! To further improve the code I have two suggestions.
- Put the value 50 in a variable called headerHeight to make the code clearer for those who read it
- Consider the size of the header as a whole, that is, use the height of the header tag (122px) instead of the height of the content (50px), as I believe this makes more sense.